op.c

```diff
@@ -3775,6 +3775,22 @@ S_optimize_op(pTHX_ OP* o)
             }
             break;
 
+        case OP_RV2AV:
+        {
+            OP *first = (o->op_flags & OPf_KIDS) ? cUNOPo->op_first : NULL;
+            if(CvSIGNATURE(PL_compcv) &&
+                    first && first->op_type == OP_GV &&
```

Comment:
same here `first && first->op_type == OP_GV` can be replaced by `OP_TYPE_IS(first, OP_GV)`

Reply:
Again.